kyuubi-server/web-ui/src/views/session/session-detail/index.vue

```diff
@@ -0,0 +1,174 @@
+<!--
+* Licensed to the Apache Software Foundation (ASF) under one
+* or more contributor license agreements.  See the NOTICE file
+* distributed with this work for additional information
+* regarding copyright ownership.  The ASF licenses this file
+* to you under the Apache License, Version 2.0 (the
+* "License"); you may not use this file except in compliance
+* with the License.  You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+-->
+
+<template>
+  <header>
+    <el-breadcrumb separator="/">
+      <el-breadcrumb-item :to="{ path: '/session/session-statistics' }">{{
+        $t('session_statistics')
+      }}</el-breadcrumb-item>
+      <el-breadcrumb-item>{{ sessionId }}</el-breadcrumb-item>
+      <el-breadcrumb-item>{{ $t('detail') }}</el-breadcrumb-item>
+    </el-breadcrumb>
+  </header>
+  <el-card
+    v-loading="sessionPropertiesLoading"
+    class="table-container session-properties-container">
+    <template #header>
+      <div class="card-header">
+        <span>{{ $t('session_properties') }}</span>
```

Comment:
let's use table instead

Reply:
Using  `el-descriptions` for set 3 col in 1 row

![popo_2023-03-27  11-26-40](https://user-images.githubusercontent.com/52876270/227833528-3c3c280c-53db-4a40-955b-1bdf0cdf9319.jpg)

Reply:
let's simply display one property in each row

Reply:
and delete the unused code instead of comment